### PR TITLE
Upgrade askomics interactive tool

### PR DESCRIPTION
**Pull request by xgaia**
ping @abretaud 

Hello,

I've upgraded the AskOmics interactive tool. It now use v19.01.3

### CODE REVIEW

**Comment by abretaud** [comment]
Forgot about this, can you fix the conflict @xgaia?

**Comment by xgaia** [comment]
Hello,
I've updated input description, and set it to optional

**Comment by xgaia** [comment]
Is it possible to have this update in the next 19.09 version once it will reviewed and merged
 ?

File: tools/interactive/interactivetool_askomics.xml
Comment (martenson) [inline tools/interactive/interactivetool_askomics.xml]: I think the version should increase.

Comment (xgaia) [reply tools/interactive/interactivetool_askomics.xml]: Done

**Comment by xgaia** [comment]
@abretaud @bgruening I've updated the tool for AskOmics 3.1.1, the latest release of AskOmics.

**Comment by nsoranzo** [comment]
@galaxybot test this

**Comment by bgruening** [comment]
failing test seems unrelated ...

**Review by martenson — APPROVED** [review]

**Comment by martenson** [comment]
@galaxybot test this

**Comment by martenson** [comment]
@bgruening @xgaia is this correct? @mvdbeek noticed

`<environment_variable name="USER_APIKEY">${__user_name__}</environment_variable>`

**Comment by bgruening** [comment]
Looks wired but up to date: https://github.com/usegalaxy-eu/galaxy/blob/release_19.09_europe/tools/interactive/interactivetool_askomics.xml


I assume the entire line is not needed.
ping @abretaud

File: tools/interactive/interactivetool_askomics.xml
Comment (abretaud) [inline tools/interactive/interactivetool_askomics.xml]: While we're at it, we should update to 3.2.0

File: tools/interactive/interactivetool_askomics.xml
Comment (abretaud) [inline tools/interactive/interactivetool_askomics.xml]: This one should be `GALAXY_API_KEY` instead of `API_KEY` (like https://github.com/usegalaxy-eu/galaxy/pull/42)

**Review by abretaud — COMMENTED** [review]
Two last minutes changes (I don't think I have write access to this PR)
For `USER_APIKEY` no idea, and @xgaia just left to vacation.. I think we can remove it and he'll make a PR later if it's needed

File: tools/interactive/interactivetool_askomics.xml
Comment (martenson) [reply tools/interactive/interactivetool_askomics.xml]: Do you know what is the `USER_APIKEY` above?

File: tools/interactive/interactivetool_askomics.xml
Comment (martenson) [reply tools/interactive/interactivetool_askomics.xml]: Does the image exist? i.e. is it only change on this line?

Comment (abretaud) [reply tools/interactive/interactivetool_askomics.xml]: yes, [askomics/flaskomics-with-dependencies:3.2.0 exists](https://hub.docker.com/layers/askomics/flaskomics-with-dependencies/3.2.0/images/sha256-c5f70e8d2ed29a96c8a825299c60063d68254d650a4bc21ea82f0bf1bc68f388), we already use it on galaxy.genouest.org

**Review by mvdbeek — APPROVED** [review]

**Comment by xgaia** [comment]
Hi all,

`<environment_variable name="USER_APIKEY">${__user_name__}</environment_variable>` is needed to log the galaxy user into AskOmics IT.

I'll do a PR to revert it.

Happy new year to everyone!

**Pull request merged**
